module-4/backend/week_two.md

```diff
@@ -4,8 +4,52 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 
 Note: When you're done, submit a PR. 
 
-TBD  
+1. What is Webpack and why is it useful?
+
+Not sure exactly. Is it a compiler of some type for front end Javascript?
+
+2. When do you want to use event delegation?
+
+I don't know
+
+3. What's one difference between ES5 and ES6?
+
+ES5 uses prototypes when creating "classes", ES6 actually uses classes that is similar to other OOP languages.
```

Comment:
ES6 utilizes prototypes as well but adds some syntactical sugar so you don't manually have to add functions to the prototype